Dockerfile

```diff
@@ -0,0 +1,10 @@
+## relay-dapp dockerfile (This file is called Dockerfile) on server side. ##
+FROM node:12-alpine AS compiler
+RUN apk add --no-cache build-base git bash
+WORKDIR /usr/src/app
+COPY package*.json ./
+COPY .env ./.env
+RUN npm i
+COPY . ./
+RUN npm start
```

Comment:
Should we adopt a multi-stage build as we did for the server?

Reply:
As this application is fairly simple for now we can leave it as it is.